### DISCUSSION TITLE
DreamBerd has been renamed to DreamBerd

### DISCUSSION THREAD

**Discussion by TodePond** [TodePond/GulfOfMexico#382] [Announcements]
DreamBerd has been through a lot of changes recently. Despite our best efforts, they haven't been popular.

**Today, we're happy to announce that DreamBerd is returning to its roots.**

No more satirical content! Just high quality programming humour for those who are smart enough.

> **"Our only priority is to make DreamBerd more popular."**

To kick things off, DreamBerd has been renamed back to DreamBerd. So...

1. Please 'unstar' [this repo](https://github.com/TodePond/WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite) immediately.
2. Please 'star' the [new repo](https://github.com/TodePond/DreamBerd) instead.

And stay tuned for more DreamBerd updates.

<hr /><em>This discussion was created from the release <a href='https://github.com/TodePond/WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite/releases/tag/v0.9z'>DreamBerd has been renamed to DreamBerd</a>.</em>

**Comment by paulaji** [comment]
finally!

**Comment by mybearworld** [comment]
Hm... I would have chosen DreamBerd instead, not DreamBerd. Still great, though

**Reply by paulaji** [reply]
Re-Re-Re-Named

**Comment by thegatesdev** [comment]
I'll just unstar ^-^

**Comment by radeeyate** [comment]
that's great! 🚀💯

**Comment by Bsodoge** [comment]
Im shaking and crying rn I cant believe this

**Comment by omril1** [comment]
What?

**Reply by mybearworld** [reply]
Dreamberd has been renamed to Dreamberd. Hope this helps :)

**Reply by omril1** [reply]
Thanks

**Comment by MarquisTheCoder** [comment]
really excited to see the future of DreamBerd from C from Dreamberd. Truly an inspiration and a shining example of transformation and growth!

**Reply by TodePond** [reply]
Don't forget WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite

**Comment by taitep** [comment]
Readme for the new repo says that it is still called C, modify it immediately.

**Reply by taitep** [reply]
In fact, you have created an infinite loop @TodePond .

**Reply by mybearworld** [reply]
These kinds of changes take time. Be patient.

**Reply by TodePond** [reply]
Everything is working as intended

**Comment by hamza-ezzahiry** [comment]
unstar star done ✅
